parachain/pallets/ethereum-beacon-light-client/Cargo.toml

```
@@ -0,0 +1,59 @@
+[package]
+name = "snowbridge-ethereum-beacon-light-client"
```

The pallet name is a bit overly verbose. Can we shorten it by removing the `light` part:

```suggestion
name = "snowbridge-ethereum-beacon-client"
```

The name of the parent folder could also be shortened to `pallets/ethereum-beacon-client`.


I've also made a similar change to our Beefy light client on Ethereum, which is now just named `BeefyClient`.